core/error_macros.h

```diff
@@ -285,7 +285,7 @@ void _err_print_index_error(const char *p_function, const char *p_file, int p_li
  * If it is null, the current function returns.
  */
 #define ERR_FAIL_NULL(m_param)                                                                          \
-	if (unlikely(!m_param)) {                                                                           \
+	if (unlikely(m_param == nullptr)) {                                                                           \
```

Comment:
You need to fix the alignment of the trailing `\`: https://travis-ci.org/github/godotengine/godot/jobs/697742242